src/tables/WebsiteAccountEntity.java

```diff
@@ -4,6 +4,8 @@
 import com.j256.ormlite.misc.BaseDaoEnabled;
```

Comment:
I can't leave a comment on app.db, but just want to be sure that nothing is diff besides you messing around w rows?